web/sass-files/sass/partials/_mentions.scss

```diff
@@ -11,13 +11,14 @@
 	position: absolute;
 	z-index: 1060;
 	.mentions-box {
-		max-height: 303px;
```

Comment:
Hey @ralder we still want a max-height on the mentions-box so that if lets say there are a lot of people in the team, the mentions-box doesn't occupy a lot of height.